src/app/content/search/reducer.ts

```diff
@@ -7,17 +7,18 @@ import { State } from './types';
 
 export const initialState = {
   loading: false,
+  open: false,
   query: null,
   results: null,
 };
 
 const reducer: Reducer<State, AnyAction> = (state = initialState, action) => {
   switch (action.type) {
     case getType(actions.requestSearch): {
-      return {...state, loading: true, query: action.payload};
+      return {...initialState, loading: true, query: action.payload, open: true};
     }
     case getType(actions.receiveSearchResults): {
-      return {...state, loading: false, results: action.payload};
+      return {...state, loading: false, results: action.payload, open: true};
```

Comment:
you shouldn't need to add `open: true` here right, because its already in `requestSearch`?

Reply:
right.